### PR TITLE
Refactor/civ 1740 remove code smells

### PR DESCRIPTION
**Pull request by taomoh**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-1740

### Change description ###

First iteration removing code smells listed here: https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&id=civil-citizen-ui

- Rename router constant to match file name (xxxController)
- Clear variable scope confusion

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by osheaf01 — APPROVED** [review]
LGTM.
Have incorporated those changes into my branch...

File: src/main/routes/routes.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import homeController from './home';
-import unauthorisedController from './unauthorised';
+import homeController from './homeController';
```
Comment (LeonardoPalmeiro) [inline src/main/routes/routes.ts:1]: There is no test into this Controller.
Maybe it's unnecessary. what do you think?

File: src/main/routes/unauthorisedController.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import {Router} from 'express';
```
Comment (LeonardoPalmeiro) [inline src/main/routes/unauthorisedController.ts:1]: There is no test into this Controller.
Maybe it's unnecessary. what do you think?

**Review by LeonardoPalmeiro — CHANGES_REQUESTED** [review]

File: src/main/routes/routes.ts (hunk 1 above)
Comment (taomoh) [reply src/main/routes/routes.ts:1]: This task is not about adding test coverage. The home page will eventually be removed anyway

File: src/main/routes/unauthorisedController.ts (hunk 2 above)
Comment (taomoh) [reply src/main/routes/unauthorisedController.ts:1]: This task is not about adding test coverage. This page could also be removed once we add standard error handling

Comment (LeonardoPalmeiro) [reply src/main/routes/unauthorisedController.ts:1]: I got you.
Thanks.

File: src/main/routes/routes.ts (hunk 1 above)
Comment (LeonardoPalmeiro) [reply src/main/routes/routes.ts:1]: thanks.

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by GorilaNaranja — APPROVED** [review]

**Review by laarmada — APPROVED** [review]

**Pull request merged**